src/SwedbankPay.Sdk.Infrastructure/JsonSerialization/JsonSerialization.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Text.Json;
+
+namespace SwedbankPay.Sdk.JsonSerialization
+{
+    public static class JsonSerialization
+    {
+        private static JsonSerializerOptions settings = null;
+
+        public static JsonSerializerOptions Settings {
+            get
+            {
+                if (settings == null)

```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)